packages/compass-e2e-tests/tests/collection-import.test.ts

```diff
@@ -95,6 +95,10 @@ describe('Collection import', function () {
     await browser.connectWithConnectionString('mongodb://localhost:27018/test');
   });
 
+  beforeEach(async function () {
+    await browser.shellEval('db.getSiblingDB("test").getCollection("json-array").deleteMany({});');
```

Comment:
So far I've been deliberately making different collections for different tests, but I can't really think of a reason why I'm doing that (other than maybe that you could examine the db after tests to debug? bit of a weak reason) so I suppose this is fine.

Reply:
I think I avoided using shellEval early on because it was very flaky at the time. Seems good now, so we should use it more.